### PR TITLE
LanguageSelection shows a clearer message if you've already tried that one

### LINKED ISSUE #122: "No results found" can be misleading
### Is your feature request related to a problem? Please describe.

The `<LanguageSelection>` component will say "No results found" if something isn't in the drop-down source. This can happen for two reasons:

* we don't have that language available (or that language doesn't exist at all)
* you've already picked the language

### Describe the solution you'd like

If you've already selected the language, the UI should reflect that, as it's otherwise confusing.

### Additional context

I'm not sure this is easily possible. `BlazoredTypeahead` just invokes `Search`, then gives that message if that returns an empty enumerable.

### PR DESCRIPTION
**Pull request by chucker**
Looks like we can fix #122 after all

### CODE REVIEW

File: Prolangle/Components/LanguageSelection.razor [hunk 1]
```diff
@@ -10,32 +10,52 @@
...
+
+		return Task.FromResult(AvailableLanguages
+			.Where(language => language.Name.Contains(Query, StringComparison.OrdinalIgnoreCase)));
+	}
 
 	[Parameter]
 	public required IReadOnlyList<ILanguage> AvailableLanguages { get; set; }
 
+	[Parameter]
+	public required IReadOnlyList<ILanguage> AlreadyPickedLanguages { get; set; }
```
Comment (ricardoboss) [inline Prolangle/Components/LanguageSelection.razor:57]: Isn't `AvailableLanguages` already filtered? Why keep track of picked languages separately?

Comment (chucker) [reply Prolangle/Components/LanguageSelection.razor:57]: Because we need the opposite here.

Comment (ricardoboss) [reply Prolangle/Components/LanguageSelection.razor:57]: Okay. Small nit: change the name to match the variables from the views (`RevealedLanguages`)

**Review by ricardoboss — APPROVED** [review]

File: Prolangle/Components/LanguageSelection.razor (hunk 1 above)
Comment (chucker) [reply Prolangle/Components/LanguageSelection.razor]: done

**Pull request merged**